lib/internal/streams/BufferList.js

```diff
@@ -2,71 +2,71 @@
 
 const Buffer = require('buffer').Buffer;
 
-module.exports = BufferList;
+module.exports = class BufferList {
```

Comment:
Yes, negligible difference in master with 5.7. Notable improvement under TF-I ... tho I'd have to pull the numbers back up. It was a week or so ago when I ran it.